AlfrescoApp/Constants/SettingConstants.h

```diff
@@ -55,7 +59,29 @@ extern NSString * const kSettingsResetAccountsIdentifier;
 extern NSString * const kSettingsResetEntireAppIdentifier;
 extern NSString * const kSettingsFullTextSearchIdentifier;
 extern NSString * const kSettingsSendFeedbackIdentifier;
-
+extern NSString * const kSettingsPasscodeLockIdentifier;
+extern NSString * const kSettingsSecurityUsePasscodeLockIdentifier;
+extern NSString * const kSettingsChangePasscodeIdentifier;
+extern NSString * const kSettingsPasscodeTouchIDIdentifier;
 extern NSString * const kSettingsSendDiagnosticsEnable;
 extern NSString * const kSettingsSendFeedbackAlfrescoRecipient;
 
+static NSUInteger const kCellLeftInset = 10;
+
+// Pin Screen strings
+extern NSString * const kSettingsSecurityPasscodeMissmatchString; // "Passcodes did not match. Try again."
```

Comment:
We generally don't add string keys as constants - is there a particular benefit to doing so?